deepchecks/core/serialization/check_failure/junit.py

```diff
@@ -0,0 +1,77 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Module containing junit serializer for the CheckFailure type."""
+import xml.etree.ElementTree as ET
```

Comment:
looks like a new dependency, how much kb does it add?

Reply:
It should be part of the standard library already.